pulp_smash/tests/rpm/api_v2/test_sync_publish.py

```diff
@@ -77,6 +79,39 @@ def _gen_distributor():
     }
 
 
+def _sync_and_publish_repo(server_config, importer=None, distributors=None):
+    """Create and sync an RPM repository.
+
+    :param server_config: A :class:`pulp_smash.config.ServerConfig` object.
```

Comment:
The `param` directive accepts either two or three arguments. Try something like this:

```
    :param pulp_smash.config.ServerConfig server_config: Information about the
        Pulp server being targeted.
```

Then run `make docs-html` and check out the result. It's pretty cool!